src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -25,10 +25,11 @@
 
 package org.spongepowered.api.world.extent;
 
-import com.flowpowered.math.vector.Vector3i;
 import org.spongepowered.api.block.BlockLoc;
 import org.spongepowered.api.world.weather.WeatherUniverse;
 
+import com.flowpowered.math.vector.Vector3i;
```

Comment:
Why not just import our code formatter config and import order? Then Eclipse should place the imports in the correct order automatically